src/breadcrumbs/breadcrumbs.scss

```diff
@@ -28,57 +28,64 @@
   // #region Component overrides
   > a {
     @include iui-anchor;
-    max-width: 24ch;
-    margin: 0 $iui-s;
+    padding: 0 $iui-s;
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
+
+    &:focus,
+    &:focus-visible {
+      outline-offset: -1px;
+    }
   }
 
   > .iui-button {
     @include iui-button-borderless;
-    --_iui-button-label-character-count: 24ch;
     border: none;
   }
 
-  &:not(.iui-current) > .iui-button {
-    --_iui-button-label-color: var(--iui-color-foreground-primary);
-    --_iui-button-label-color-hover: var(--_iui-button-label-color);
+  > * {
+    max-width: 26ch;
+  }
+
+  &:not(.iui-current) > * {
+    &,
+    &:hover,
+    &:active {
+      --_iui-button-foreground-color: var(--iui-color-foreground-primary);
+    }
```

Comment:
This is not nice. If I remove `:hover` and `:active`, then it looks like they have the same specificity but `iui-button` rules are declared later in the stylesheet so they take preference
![image](https://user-images.githubusercontent.com/9084735/163416657-b2dbf07c-34f9-461d-9e7c-ed672f030a3d.png)

I think in the future we might want to use `:where` for the hover/active states in base button. For now this is fine.

Reply:
I have an idea on how to solve this.  Instead of setting the button color & button hover color with two separate RGBA, what if we set the just the button color and change the button label opacity instead?  The added benefit is it would be easier for others to override the button color, needing to only set 1 color instead of 2.

Reply:
Yeah you could go that route or leave it as is. I'm fine either way, as it's a button issue, not a breadcrumb issue.